lib/Service/NoteUtil.php

```diff
@@ -183,6 +183,13 @@ public function getOrCreateFolder(string $path, bool $create = true) : Folder {
 		return $folder;
 	}
 
+	public function getNotesFolderUserPath(string $userId): ?string {
+		/** @psalm-suppress MissingDependency */
+		$userFolder = $this->getRoot()->getUserFolder($userId);
+		$nodesFolder = $this->getOrCreateNotesFolder($userId, false);
```

Comment:
This is probably on users where the folder does not exists yet (e.g. new users on first login). 
Possible solutions: either the `create` parameter has to be true, or the `NotesFolderException` has to be caught.

Reply:
An integration test would have caught this btw.